packages/card/index.js

```diff
@@ -1,17 +1,45 @@
 import { html, css } from 'lit';
 
+import { classNames } from '@chbphone55/classnames';
+import { i18n } from '@lingui/core';
 import { card as ccCard } from '@warp-ds/css/component-classes';
 import WarpElement from '@warp-ds/elements-core';
 import { ifDefined } from 'lit/directives/if-defined.js';
 
-import { fclasses } from '../utils';
+import { activateI18n } from '../i18n';
+import { kebabCaseAttributes } from '../utils';
+
+import { messages as daMessages } from './locales/da/messages.mjs';
+import { messages as enMessages } from './locales/en/messages.mjs';
+import { messages as fiMessages } from './locales/fi/messages.mjs';
+import { messages as nbMessages } from './locales/nb/messages.mjs';
 
 const keys = {
   ENTER: 'Enter',
   SPACE: ' ',
 };
 
-class WarpCard extends WarpElement {
+class WarpCard extends kebabCaseAttributes(WarpElement) {
```

Comment:
This wrapper is redundant, because this web component only expects single-word attributes.

Reply:
Updated in this commit: [ab72db0](https://github.com/warp-ds/elements/pull/177/commits/ab72db064323acb00daf8cf2d23036e395021ee8)